src/lib/authserver/dir/ds389.rb

```diff
@@ -9,20 +9,20 @@
 # this program; if not, contact SUSE LINUX GmbH.
 
 # Authors:      Howard Guo <hguo@suse.com>
+#               William Brown <wbrown@suse.de>
```

Comment:
btw we are slightly moving away from this practice as git blame or git log or github contributors already provide this info.